examples/example_create_lambda_function/main.tf

```diff
@@ -0,0 +1,20 @@
+module "landing_zone_reader" {
+  source = "../../modules/landing_zone_reader"
+}
+
+resource "aws_lambda_function" "my_demo" {
+  function_name = "My_Lambda_Demo"
+  description   = "Managed by TerraHub"
+  handler       = "handler.index"
+  memory_size   = "128"
+  role          = "arn:aws:iam::123456789012:role/ServiceRoleForLambdaEdge"
```

Comment:
this is ok ... for now. in future, please rewrite this line to reference another output from `landing_zone_reader` ;)